cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -1,119 +1,600 @@
+    # except Exception as e:
+    #     frappe.log_error(f"Error in get_salary_component: {str(e)}")
+    #     return {"error": str(e)}
+
+
+
+
+import frappe
+import json
+
+@frappe.whitelist()
+def get_salary_component(data, component, custom_field=None):
```

Ruff (0.8.2)

`get_salary_component` is too complex (15 > 10)

(C901)